### DISCUSSION TITLE
quarkus-amazon-lambda vs funqy

### DISCUSSION THREAD

**Discussion by ootsun** [quarkusio/quarkus#34295] [Q&A]
Hi,
I want to build a serverless application and deploy it on AWS.
If I don't need to be able to switch from cloud provider, is there a reason to go with Funqy instead of quarkus-amazon-lambda?

[a comment by an automated account is not shown]

**Comment by patriot1burke** [comment]
Correct. No reason to use Funqy if you don't want a function layer that's portable between function providers.
